pkg/apis/monitoring/v1alpha1/alertmanager_config_types.go

```diff
@@ -90,13 +90,16 @@ type AlertmanagerConfigSpec struct {
 	// +optional
 	Route *Route `json:"route"`
 	// receivers defines the list of receivers.
+	// +listType=atomic
```

Comment:
Alertmanager expects receiver's names to be unique so we can use `+listType=map`.